src/reducer_modal.js

```diff
@@ -0,0 +1,13 @@
+import * as actions from './actions';
+
+export function modal(state=null, action) {
+  switch (action.type) {
+    case actions.SHOW_MODAL:
+      return action.payload
+
+    case actions.HIDE_MODAL:
+      return null
+    default:
+      return state
+  }
+}
```

Comment:
modal isn't a good name for this reducer. It's only concerned with the selectedUser and is even called that in the root reducer and named that way in the redux store. You should call it selectedUser. The reducers should be named after the part of the store they are responsible for.